src/utils/soknad-felles/hentSoknadStatustekst.js

```diff
@@ -27,9 +27,8 @@ const textSendtTilArbeidsgiver = (
 };
 
 const hentStatustekst = (soknad) => {
-  const soknadSendtTilNav =
-    soknad.sendtTilNAVDato !== null || soknad.innsendtDato !== null;
-  const soknadSendtTilArbeidsgiver = soknad.sendtTilArbeidsgiverDato !== null;
+  const soknadSendtTilNav = !!soknad.sendtTilNAVDato || !!soknad.innsendtDato;
+  const soknadSendtTilArbeidsgiver = !!soknad.sendtTilArbeidsgiverDato;
```

Comment:
Er det mulig å prefikse disse variablene med er eller is, så vi gjør det ekstra tydelig at de holder på boolske verdier?